dashboard/app/views/plc/enrollment_task_assignments/complete/_script_completion_task.html.haml

```diff
@@ -0,0 +1,9 @@
+Hello, this is a script you have to complete.
+
+- script = @task.script_id ? Script.find(@task.script_id) : nil
```

Comment:
This seems to be asking to be turned into `@task.script` (or `@task.try(:script)` if not all tasks should respond to `script`)